### ISSUE TITLE
[Bug] Rule node still work after deleted

### ISSUE THREAD

**Issue by xiddjp** [thingsboard/thingsboard#6379]
**Describe the bug**
In my rule chain, I added a rule node(such as mqtt rule node).
First it works well, and i deteted this node, then i restart tb-core rule-engine services.
But i still see the mqtt actions in the log. Why?

**Your Server Environment**
  * ThingsBoard Version: 3.3.2
  * OS Name and Version: ubuntu18

**Comment by Backdraft007** [comment]
Can you specify the details a little bit more?
Are the mqtt actions from TB-Server to the broker? Or are there other mqtt actions?

**Comment by xiddjp** [comment]
@Backdraft007 , I added a custom rule node(to create device from telemetry) to my rule chain.
First it works well, Now I don't need this rule node. So I deleted this rule node, and saved my rule chain.
But after I restart the tb-node and tb-core containers, I find that there will still be devices created through the previously deleted rule nodes.
I have checked many times to ensure that the rule nodes in the rule chain have been deleted. This situation puzzles me. Why do the deleted rule nodes continue to work?
